### PR TITLE
feat() Add Hadoop example config for google exporter

### PR DESCRIPTION
**Pull request by Mrod1598**
Added Hadoop example config for Google Cloud Monitoring.

image

##### Checklist
- [x] Changes are tested
- [ ] CI has passed

### CODE REVIEW

**Review by cpheps — COMMENTED** [review]
Has this been tested ti verify it works?

File: config/google_cloud_exporter/hadoop/config.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      value: "global"
+      action: upsert
+
+  normalizesums:
+
+  batch:
+
+exporters: 
+  logging:
+    loglevel: debug
```
Comment (BinaryFissionGames) [inline config/google_cloud_exporter/hadoop/config.yaml:38]: This logging exporter should be removed

File: config/google_cloud_exporter/hadoop/config.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    - key: namespace
+      value: "hadoop"
+      action: upsert
+    - key: location
+      value: "global"
+      action: upsert
+
+  normalizesums:
+
+  batch:
```
Comment (BinaryFissionGames) [inline config/google_cloud_exporter/hadoop/config.yaml:34]: These (normalize sums, batch) are built into the version of `googlecloud` exporter we now use, which is a wrapped version of the google exporter. (https://github.com/observIQ/observiq-otel-collector/tree/main/exporter/googlecloudexporter)

Comment (BinaryFissionGames) [inline config/google_cloud_exporter/hadoop/config.yaml:34]: These are built into the version of `googleexporter` we now use, which is a wrapped version of the google exporter. (https://github.com/observIQ/observiq-otel-collector/tree/main/exporter/googlecloudexporter)

**Review by BinaryFissionGames — COMMENTED** [review]
I think we should be writing these for our wrapped google exporter from now on, unless we have a reason not to.

**Comment by BinaryFissionGames** [comment]
Seems like we're actually keeping these configs with the current exporter that's released.

File: config/google_cloud_exporter/hadoop/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Tomcat Metrics with Google Cloud
```
[the remark this answers is not in the document]
Comment (Mrod1598) [reply config/google_cloud_exporter/hadoop/README.md]: Whoops, Updated. Also, added example to description

File: config/google_cloud_exporter/hadoop/config.yaml (hunk 1 above)
Comment (Mrod1598) [reply config/google_cloud_exporter/hadoop/config.yaml]: Removed

**Review by BinaryFissionGames — APPROVED** [review]

**Review by cpheps — APPROVED** [review]

**Pull request merged**